Vic2ToHoI4/Source/HOI4World/Map/SupplyNodes.cpp

```diff
@@ -0,0 +1,26 @@
+#include "SupplyNodes.h"
+
+
+
+std::set<int> HoI4::determineSupplyNodes(const std::map<int, std::shared_ptr<Vic2::Province>>& Vic2Provinces,
+	 const Mappers::ProvinceMapper& provinceMapper)
+{
+	std::set<int> supplyNodes;
+	for (const auto& [Vic2ProvinceNumber, Vic2Province]: Vic2Provinces)
+	{
+		if (!Vic2Province->isLandProvince())
+		{
+			continue;
+		}
+
+		const auto HoI4ProvinceNumbers = provinceMapper.getVic2ToHoI4ProvinceMapping(Vic2ProvinceNumber);
+		if (HoI4ProvinceNumbers.empty())
+		{
+			continue;
+		}
+
```

Comment:
iterate through hoi4 provinces and see if they contain railways. You can use their presence to clamp down on supply depo locations.

Reply:
That's the plan for #1300